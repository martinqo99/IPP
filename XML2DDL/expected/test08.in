<?xml version="1.0"?>
<catalog>
  <part number="1">
   <alphabet a="True" b="True" c="True"/>
   <alphabet c="True" d="True" e="True"/>
   <alphabet f="True" g="True" a="True"/>
   <alphabet H="True" h="True" i="True"/>
   <alphabet j="True" k="True"/>
   <alphabet l="True" L="True" m="True"/>
  </part>
  <part number="2">
   <alphabet n="True" o="True" p="True"/>
   <alphabet q="True" r="True" s="True"/>
   <alphabet u="True" t="True" s="True"/>
   <alphabet v="True" w="True" a="True"/>
   <alphabet b="True" c="True"/>
   <alphabet z="True" a="True" f="True"/>
  </part>
</catalog>
