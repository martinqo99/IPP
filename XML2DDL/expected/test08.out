--Takto pak vypadá hlavička výstupního souboru

CREATE TABLE alphabet(
   prk_alphabet_id INT PRIMARY KEY,
   part_id INT,
   a BIT,
   c BIT,
   b BIT,
   e BIT,
   d BIT,
   g BIT,
   f BIT,
   i BIT,
   h BIT,
   k BIT,
   j BIT,
   m BIT,
   l BIT,
   o BIT,
   n BIT,
   q BIT,
   p BIT,
   s BIT,
   r BIT,
   u BIT,
   t BIT,
   w BIT,
   v BIT,
   z BIT
);

CREATE TABLE part(
   prk_part_id INT PRIMARY KEY,
   number INT
);

